plex/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.7"
 
 services:
   server:
-    image: linuxserver/plex:version-1.32.5.7349-8f4248874@sha256:4f79fb474e01c629301c90bd3f39d47cc83678e6aceda15effa3e6320f8e296b
+    image: linuxserver/plex:1.32.6@sha256:6b82fe376f675a55204b68bd66a369d56c46eccc2fbb34fededa538c1a460b67
```

Comment:
It looks like the latest tag is this one here:
<img width="619" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/d0453930-c49d-4f38-963b-431008a00c05">

Which is here on docker hub: https://hub.docker.com/layers/linuxserver/plex/1.32.6.7557-1cf77d501-ls188/images/sha256-d125d36cc26129936e429b4c9d0964a729a9a9d21ee5f9a03ac73dd6b7137981?context=explore

This one is nice to use because it matches the current Plex Media Server version available for download (1.32.6.7557), and we can just copy the lastest Plex Media release notes (shown in a comment below).

Reply:
Hey @nmfretz, Both images have same digest. I used `1.32.6` because it looks better 😄 . Anyways, I changed it to use the specific version you specified.